### DISCUSSION TITLE
Base64: invalid input

### DISCUSSION THREAD

**Discussion by Umarajaa4** [justchokingaround/lobster#147] [Q&A]
Lobster was running fine before but today when i try to  play any movie it gives the error mentioned in the heading. I've tried updating lobster and have updated curl too but still no solution. Any help would be appreciated. TIA

**Comment by justchokingaround** [comment]
tldr: flixhq (or rather rapidstream and dokicloud) changed their way of obfuscating the js, so getting the decryption keys isnt possible atm, working in it

you can follow issue #144 for most recent updates

**Reply by justchokingaround** [reply]
https://github.com/justchokingaround/lobster/commit/dfaf1a3a93f6793127c7cbea20759ef7333a662c should fix this
